packages/extension-polymath-ui/src/Popup/Signing/index.tsx

```diff
@@ -2,20 +2,26 @@
 // This software may be modified and distributed under the terms
```

Comment:
Changes to this entire file are probably out of sync. I had synced that file with the latest code from Polkadot yesterday. And I don't see any polymath-specific changes here. We should probably revert all changes in this file.